website/docs/docs/build/documentation.md

```diff
@@ -2,15 +2,18 @@
 title: "Documentation"
```

Comment:
Should we change the sidebar title to "About documentation" or something similar to further distinguish it from the "View documentation" option on the sidebar?

Reply:
yea good shout!